sbi/inference/trainers/fmpe/fmpe.py

```diff
@@ -98,6 +99,14 @@ def build_posterior(
             Posterior $p(\theta|x)$  with `.sample()` and `.log_prob()` methods.
         """
 
+        warn_if_deprecated(
+            self.build_posterior,
+            locals(),
+            {
+                "vectorfield_sampling_parameters",
+            },
```

Comment:
formatting is a bit verbose here, can we reduce the number of lines by removing the trailing commas? If no, it's fine.